packages/js/product-editor/src/components/product-mvp-feedback-modal/product-mvp-feedback-modal.tsx

```diff
@@ -100,45 +109,63 @@ function ProductMVPFeedbackModal( {
 					weight="600"
 					size="14"
 					lineHeight="20px"
-				>
-					{ __(
-						'What made you switch back to the classic product editor?',
-						'woocommerce'
-					) }
-				</Text>
-				<Text
-					weight="400"
-					size="12"
-					as="p"
-					lineHeight="16px"
-					color="#757575"
-					className="woocommerce-product-mvp-feedback-modal__subtitle"
-				>
-					{ __( '(Check all that apply)', 'woocommerce' ) }
-				</Text>
-				<div className="woocommerce-product-mvp-feedback-modal__checkboxes">
-					{ checkboxes.map( ( checkbox, index ) => (
-						<CheckboxControl
-							key={ index }
-							label={ checkbox.label }
-							name={ checkbox.key }
-							checked={ checkbox.checked }
-							onChange={ checkbox.onChange }
-						/>
-					) ) }
-				</div>
+				></Text>
+				<fieldset className="woocommerce-product-mvp-feedback-modal__reason">
+					<legend>
```

Comment:
Can you increase the margin bottom here? It is very close to the checkboxes and the margin is bigger in the design

Reply:
Nice catch!  Updated from `$gap-smaller` to `$gap-small`.